### PR TITLE
remove installation of packages in nb sync

### PR DESCRIPTION
**Pull request by KPostOffice**
# What changes have been made
remove line doing CF-SDK installation just lock with new dependency

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: .github/workflows/odh-notebooks-sync.yml [hunk 1]
```diff
@@ -62,10 +62,8 @@ jobs:
               mv $1 Pipfile
               mv $2 Pipfile.lock
             fi
-            if ! pipenv install ${package_name}~="${CODEFLARE_RELEASE_VERSION}"; then
-              echo "Failed to install ${package_name} with version ${CODEFLARE_RELEASE_VERSION} in $dir"
-              exit 1
-            fi
+            # replace existing version of cf-sdk with new version in Pipfile
+            echo "$(sed 's/codeflare-sdk = .*$/codeflare-sdk = \"~=$CODEFLARE_RELEASE_VERSION\"/g' Pipfile | envsubst)" > Pipfile
```
Comment (jstourac) [inline .github/workflows/odh-notebooks-sync.yml:66]: Maybe I'm missing something, but why not to use simply the `sed -i ...` ? E.g.:
```suggestion
            sed --in-place 's/codeflare-sdk = .*$/codeflare-sdk = "~=${CODEFLARE_RELEASE_VERSION}"/g' Pipfile
```
:thinking:

Comment (KPostOffice) [reply .github/workflows/odh-notebooks-sync.yml:66]: Thanks @jstourac, no reason not to. My sed foo is lacking

Comment (KPostOffice) [reply .github/workflows/odh-notebooks-sync.yml:66]: Jk, this doesn't seem to work

**Review by jstourac — APPROVED** [review]
/lgtm

File: .github/workflows/odh-notebooks-sync.yml (hunk 1 above)
Comment (jstourac) [reply .github/workflows/odh-notebooks-sync.yml]: Right, sorry, I didn't realize :slightly_smiling_face: I put it here without any testing. Thanks for the change!

**Review by Bobbins228 — APPROVED** [review]
/approve

**Comment by jstourac** [comment]
Just realized - without this, the Pipfile.lock isn't updated, is it? :thinking:  Didn't check whether there is some call elsewhere in the file now.

**Pull request merged**